app/code/Magento/Wishlist/Model/Rss/Wishlist.php

```diff
@@ -272,7 +283,9 @@ public function getFeeds()
     }
 
     /**
-     * {@inheritdoc}
+     * Is Auth Required
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc